db/elasticsearch/migrations/1.0.1/down/03-c-index-content.json

```diff
@@ -9,7 +9,7 @@
     "properties": {
       "id": { "type": "long" },
       "uid": { "type": "text" },
-      "sourceId": { "type": "integer", "index": false },
+      "sourceId": { "type": "integer" },
```

Comment:
This is a rollback script so this change is not necessary.

Reply:
Updated, thanks